.circleci/config.yml

```
@@ -2,10 +2,39 @@ version: 2.1
 orbs:
   elixir: membraneframework/elixir@1
 
+jobs:
+  test_with_db:
+    docker:
+      - image: membraneframeworklabs/docker_membrane:latest
+        environment:
+          MIX_ENV: test
+      - image: timescale/timescaledb:2.5.1-pg14
+        command: [-cshared_preload_libraries=timescaledb]
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: membrane_rtc_engine_timescaledb_test  
+    steps:
+      - checkout
+      - run: mix deps.get
```

You should base on the `test` task from the orb https://github.com/membraneframework/circleci-orb/blob/master/src/jobs/test.yml
That means you should use commands with caching
```suggestion
      - elixir/get_mix_deps
      - elixir/use_build_cache:
          env: test
```